assignments/prototypes.js

```diff
@@ -41,11 +70,11 @@
 
 // Test you work by un-commenting these 3 objects and the list of console logs below:
 
-/*
```

Comment:
Don't forget to take out the ending part of this comment, which is in line 134, when you're uncommenting code.